src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx

```diff
@@ -69,7 +69,7 @@ export class NodeContextMenu extends React.PureComponent<Props> {
         }
         break;
       case NodeType.SERVICE:
-        type = node.isServiceEntry ? Paths.SERVICEENTRIES : Paths.SERVICES;
+        type = node.seInfo ? Paths.SERVICEENTRIES : Paths.SERVICES;
```

Comment:
For consistency, please change this back to isServiceEntry (as mentioned on the server PR).